### ISSUE TITLE
SQL Database Projects: SqlStaticCodeAnalysisTask task was not found when building project

### ISSUE THREAD

**Issue by holytshirt** [microsoft/azuredatastudio#13781]
- Azure Data Studio Version:
1.25.1
1.26.0 insiders

Steps to Reproduce:
1. Opened Sql Database project created in Visual Studio with "Enable Code Analysis on Build" Enabled.  
Build Fails with
``` 
stdout: c:\Users\blah\.azuredatastudio-insiders\extensions\microsoft.sql-database-projects-0.5.1\BuildDirectory\Microsoft.Data.Tools.Schema.SqlTasks.targets(1345,7):
error MSB4036: The "SqlStaticCodeAnalysisTask" task was not found. Check the following: 
1.) The name of the task in the project file is the same as the name of the task class. 
2.) The task class is "public" and implements the Microsoft.Build.Framework.ITask interface. 
3.) The task is correctly declared with <UsingTask> in the project file, or in the *.tasks files located in the "C:\Program Files\dotnet\sdk\5.0.100" directory. [d:\dev\blah\blah\src\Database\Database.sqlproj]
```

2. Opened Sql Database project created in Visual Studio with "Enable Code Analysis on Build" Disabled.
Build is successful!

**Comment by kisantia** [comment]
needs to be done in DacFx

**Comment by zijchen** [comment]
This should be resolved in SDK version > 0.2.3-preview
